src/shared/services/email/email.service.ts

```diff
@@ -20,35 +16,47 @@ const transporter = createTransport({
   },
 });
 
+const getStringFile = async (): Promise<string> => {
+  const filePath = path.resolve(process.cwd(), 'html/email-index.html');
+
+  const stringFile = await new Promise<string>((res) => {
+    fs.readFile(filePath, async (err, data) => {
+      if (err) {
+        throw err;
+      }
+
+      res(data.toString());
+    });
+  });
+
+  return stringFile;
+};
+
+const stringFile = getStringFile();
+
+const getEmailHTML = async <T extends object>(templateContext: T): Promise<string> => {
+  const handlebarFileString = await stringFile;
+
+  const template = handlebars.compile(handlebarFileString);
+
+  return template(templateContext);
+};
+
 export const sendMessageToEmail = async (
   toEmail: string,
   code: string,
-  type: string
+  type: EmailEnum
 ): Promise<void> => {
-  const sendData: ISendData = {
-    from: `Kravich13 <${smtpConfig.user}>`,
+  const actionTypeTitle = type === EmailEnum.REGISTRATION_LETTER ? 'registration' : 'recovery';
```

Comment:
Why this way? 
1. You started this expression with enums but finished with hardcoded strings
2. Don't bind your logic this way: what if you'll have more than 2 types of letters? Just use enums and that's it